<launch>
	<!-- rviz -->
	<node pkg="rviz" type="rviz" name="rviz" args="-d $(find dummy_msg_publication)/rviz_config/pointcloud2.rviz"/>

	<!-- main -->
	<node pkg="dummy_msg_publication" type="pointcloud2" name="pointcloud2" output="screen">
		<!-- <param name="hz" type="double" value="10.0"/> -->
		<!-- <param name="frame" type="string" value="frame"/> -->
		<!-- <param name="rand_range" type="double" value="100.0"/> -->
		<!-- <param name="num_points" type="int" value="1000"/> -->
	</node>
</launch>